components/script/dom/bluetooth.rs

```diff
@@ -49,9 +52,77 @@ const NAME_PREFIX_ERROR: &'static str = "'namePrefix', if present, must be nonem
 const NAME_TOO_LONG_ERROR: &'static str = "A device name can't be longer than 248 bytes.";
 const SERVICE_DATA_ERROR: &'static str = "'serviceData', if present, must be non-empty to filter devices.";
 const SERVICE_ERROR: &'static str = "'services', if present, must contain at least one service.";
-const OPTIONS_ERROR: &'static str = "Fields of 'options' conflict with each other.
+pub const OPTIONS_ERROR: &'static str = "Fields of 'options' conflict with each other.
```

Comment:
This const is used in another [file](https://github.com/szeged/servo/pull/92/files/0affd1aa099303de4aa8b3e1c55e32017e706566#diff-f978d59fbb580e780099d649ab652468R18) to.
Should I redeclare this const in that to?